src/Catty/Supporting Files/AppDelegate.swift

```diff
@@ -30,8 +32,11 @@ func uncaughtExceptionHandler(exception: NSException) {
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
     var window: UIWindow?
+    var firebaseAnalyticsReporter: FirebaseAnalyticsReporter?
+    var firebaseCrashlyticsReporter: FirebaseCrashlyticsReporter?
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        self.setupFirebase()
```

Comment:
Please cluster both setup method (e.g., move it back to the `setupSiren` or move the `setupSiren` method above as well).